src/components/backend-ai-settings-view.ts

```diff
@@ -433,27 +551,202 @@ export default class BackendAiSettingsView extends BackendAIPage {
-        this.notification.text = _text('settings.JobSchedulerUpdated');
+   * find id in html by scheduler option key.
+   * 
+   * @param option - scheduler option key 
+   * @returns id
+   */
+  _findIdByOption(option) {
+    return this.schedulerOptionsAndId.find(elm => elm.option === option).id;
+  }
+
+  /**
+   * find scheduler option key by id in html.
+   * 
+   * @param id - id in html
+   * @returns option
+   */
+  _findOptionById(id) {
+    return this.schedulerOptionsAndId.find(elm => elm.id === id).option;
+  }
+
+  /**
+   * Clear option values from environment container.
+   */
+  _clearOptions() {
+    const container = this.shadowRoot.querySelector('#env-container');
+    // initialize options (textfield values)
+    container.querySelectorAll('mwc-textfield').forEach((tf) => {
+      tf.value = '';
+    });
+  }
+
+  _showEnvDialog() {
+    const envDialog = this.shadowRoot.querySelector('#env-dialog');
+    envDialog.closeWithConfirmation = true;
+    envDialog.show();
+  }
+
+  _hideEnvDialog() {
+    const envDialog = this.shadowRoot.querySelector('#env-dialog');
+    envDialog.closeWithConfirmation = false;
+    envDialog.hide();
+  }
+
+  /**
+   * Close confirmation dialog and environment variable dialog and reset the option values.
+   */
+  closeAndResetEnvInput() {
+    const schedulerSwitch = this.shadowRoot.querySelector('#scheduler-switch');
+    schedulerSwitch.value = null;
+    this._clearOptions();
+    this.closeDialog('env-config-confirmation');
+    this._hideEnvDialog();
+  }
+
+  _showConfigDescription(e, item) {
+    e.stopPropagation();
+    const schedulerConfigDescription = {
+      'default': {
+        'title': _tr('settings.ConfigPerJobSchduler'),
+        'desc': _text('settings.ConfigPerJobSchdulerDescription')
+      },
+      'retries': {
+        'title': _text('settings.SessionCreationRetries'),
+        'desc': _text('settings.SessionCreationRetriesDescription') + '\n' + _text('settings.FifoOnly')
+      }
+    };
+    if (item in schedulerConfigDescription) {
+      this._helpDescriptionTitle = schedulerConfigDescription[item].title;
+      this._helpDescription = schedulerConfigDescription[item].desc;
+      const desc = this.shadowRoot.querySelector('#help-description');
+      desc.show();
+    }
+  }
+
+  openDialog(id) {
+    this.shadowRoot.querySelector('#' + id).show();
+  }
+
+  closeDialog(id) {
+    this.shadowRoot.querySelector('#' + id).hide();
+  }
+
+  /**
+   * Update etcd scheduler options and close the dialog.
+   * 
+   */
+  saveAndCloseDialog() {
+    const scheduler = this.shadowRoot.querySelector('#scheduler-switch');
+    const numRetriesPerSchedulerEl = this.shadowRoot.querySelector('#num-retries');
+    const tempNumRetries = numRetriesPerSchedulerEl.value;
+    const inputValidationArr = [scheduler, numRetriesPerSchedulerEl];
+    if (inputValidationArr.filter(elem => elem.reportValidity()).length < inputValidationArr.length) {
```

Comment:
@Sujin-Kim1 
Although we can use snackbar UI on the right bottom of the screen when showing input is invalid,
but If the component supports validation message right below the input field,
then we could rather use it instead of showing an error message with snackbar.
Since both mwc-select and mwc-textfield component supports validation message area,
I updated validity checking in both input fields(including select UI) before requesting configuration setting to the manager server.